java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4961,6 +4961,17 @@ void testSetOperations() {
     }
   }
 
+  @Test
+  void testReverseString() {
+    try (ColumnVector input = ColumnVector.fromStrings("abcdef", "12345", "", "", "aébé",
```

Comment:
maybe a non-issue but it is probably safer to write non-ascii strings as unicode escapes "\uHEX" to avoid dealing with editor depending on the region.

Reply:
This is actually not an issue here because libcudf code (in the same repo) has a lot of tests with such non-ascii characters.